cvat/settings/base.py

```diff
@@ -192,10 +192,15 @@ def generate_ssh_keys():
 CORS_ALLOW_CREDENTIALS = True
 CSRF_TRUSTED_ORIGINS = [UI_HOST]
 UI_URL = '{}://{}'.format(UI_SCHEME, UI_HOST)
+UI_URL_WITHOUT_PORT = UI_URL
+
+
 if len(UI_URL):
     UI_URL += ':{}'.format(UI_PORT)
 
-CORS_ORIGIN_WHITELIST = [UI_URL]
+# Add UI_URL_WITHOUT_PORT to avoid CORS Policy, ERR_CONNECTION_REFUSED
+# errors when UI_URL is <FQDN>:80
+CORS_ORIGIN_WHITELIST = [UI_URL, UI_URL_WITHOUT_PORT]
```

Comment:
Do you think we should have `if UI_PORT == 80` here?

Reply:
Yes, it would be cleaner.  Now I'm deploying again with the if statement and testing the issue to assert that the problem is still fixed. Otherwise, it doesn't hurt to always have the UI_URL_WITHOUT_PORT variable in the CORS_ORIGIN_WHITELIST.

Reply:
I believe that the UI_URL should be added to the CORS_ORIGIN_WHITELIST with and without the port number regardless of which port it is, because the application port can be 8080 or any other even when connecting via FQDN. I'd say this PR fix is suitable to be accepted